dev-docs/bidders/adform.md

```diff
@@ -4,11 +4,12 @@ title: Adform
 description: Prebid Adform Bidder Adaptor
 hide: true
 biddercode: adform
-media_types: video
+media_types: banner, video
 gdpr_supported: true
 usp_supported: true
 tcf2_supported: true
 prebid_member: true
+userIds: pubCommonId, unifiedId, id5Id, parrableId, identityLink, liveIntentId, britepoolId, digitrust, criteo, netId, sharedId
```

Comment:
we've been using alphabetically order to maintain consistency between adapters:

```suggestion
userIds: britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, sharedId, unifiedId
```

also note that digitrust is being removed as of Prebid 4.0, so up to you if you leave that one in for now or not.